README.md

```diff
@@ -98,4 +98,19 @@ func consumeFn(message kafka.Message) error {
 | `sasl.authType`                    | `SCRAM` or `PLAIN`                                                                                                                    |                  |
 | `sasl.username`                    | SCRAM OR PLAIN username                                                                                                               |                  |
 | `sasl.password`                    | SCRAM OR PLAIN password                                                                                                               |                  |
-| `logger`                           | If you want to custom logger                                                                                                          | info             |                          |
+| `logger`                           | If you want to custom logger                                                                                                          | info             |
+| `apiEnabled`                       | Enabled metrics                                                                                                                       | false            |
+| `apiConfiguration.port`            | Set API port                                                                                                                          | 8090             |
```

Comment:
8080 is a docker-compose kowl port, so I needed to specify a different port, like 8090.